pkg/controller/endpointslice/endpointslice_controller_test.go

```diff
@@ -24,6 +24,7 @@ import (
 	"testing"
 	"time"
 
+	"github.com/benbjohnson/clock"
```

Comment:
This dependency does not currently exist in the kubernetes tree... is this PR part of some larger project? Does it depend on some other PR?